docker/docker-compose.yml

```diff
@@ -15,15 +39,20 @@ services:
       context: ../
       dockerfile: docker/girder.Dockerfile
     image: kitware/viame-web:${TAG:-latest}
+    healthcheck:
+        test: ["CMD", "curl", "-f", "http://localhost:8080"]
+        interval: 5s
+        timeout: 5s
+        retries: 5
```

Comment:
Why these specific parameters, or why the health check at all? 

I don't really have a good answer for the first question (just kind of chose some), but the reason it's there at all is so that the worker images aren't started immediately after the girder image. Since the girder image actually installs girder, and _then_ runs `girder serve`, the workers can start before it, and try to re-run existing tasks, failing them because girder isn't running.

Reply:
Excellent, thanks.  I started this wiki page just to keep track of random stuff like this.

https://github.com/VIAME/VIAME-Web/wiki/Miscellaneous-Server-Operator-notes.